crates/ruma-client-api/src/discovery/discover_homeserver.rs

```diff
@@ -104,3 +111,21 @@ impl TileServerInfo {
         Self { map_style_url }
     }
 }
+
+/// Information about a discovered authentication server.
+#[cfg(feature = "unstable-msc2965")]
+#[derive(Clone, Debug, Deserialize, Hash, Serialize)]
+#[cfg_attr(not(feature = "unstable-exhaustive-types"), non_exhaustive)]
+pub struct AuthenticationServerInfo {
+    /// The base URL for the identity server for client-server connections.
+    pub issuer: String,
+    pub account: Option<String>,
```

Comment:
Please document this field as well.

Reply:
Ah my bad, I did actually notice a missing docs warning now I think about it, but wasn't sure where it was referencing even though it was in plain sight 🙈

Reply:
Updated as a separate commit, assuming you'll squash when it comes time to merge :)